### PR TITLE
options: don't include "options" in 'sessionoptions' by default.

### PR DESCRIPTION
**Pull request by fmoralesc**
Re: https://github.com/neovim/neovim/issues/2676

### CODE REVIEW

**Comment by Pyrohh** [comment]
There's a merge conflict, and could you also `:sort` the list in `vim_diff.txt`? It seems to have gotten un-alphabetized recently.

**Comment by fmoralesc** [comment]
:+1: Done.

File: runtime/doc/options.txt [hunk 1]
```diff
@@ -5406,6 +5406,8 @@ A jump table for the options with a short description can be found at |Q_op|.
 
 						*'sessionoptions'* *'ssop'*
 'sessionoptions' 'ssop'	string	(default: "blank,buffers,curdir,folds,
+					       help,tabpages,winsize"
+				 Vi default: "blank,buffers,curdir,folds,
 					       help,options,tabpages,winsize")
 			global
 			{not available when compiled without the |+mksession|
```
Comment (Pyrohh) [inline runtime/doc/options.txt]: might as well remove this annotation while we're here

**Comment by Pyrohh** [comment]
> Done

Thanks. Besides the nit I just mentioned, LGTM.

File: runtime/doc/options.txt (hunk 1 above)
Comment (fmoralesc) [inline runtime/doc/options.txt]: Done.

**Comment by justinmk** [comment]
Merged

**Pull request closed without merging**